Source/WebCore/platform/PlatformEvent.h

```diff
@@ -155,4 +155,45 @@ template<> struct EnumTraits<WebCore::PlatformEvent::Modifier> {
     >;
 };
 
+template<> struct EnumTraits<WebCore::PlatformEvent::Type> {
+    using values = EnumValues<
+        WebCore::PlatformEvent::Type,
+        WebCore::PlatformEvent::Type::NoType,
+
+            // PlatformKeyboardEvent
```

Comment:
Nit: Please fix indentation of the comment to match the enum values here and below.